app/assets/stylesheets/story_show_page.scss

```diff
@@ -1,3 +1,7 @@
 .personal-image {
   margin-bottom: 2rem;
 }
+
+.simple_form input[type='submit'] {
```

Comment:
Avoid qualifying attribute selectors with an element.<br>Selector `simple_form` should be written in lowercase with hyphens